packages/import-utils/src/create-sandbox/index.ts

```diff
@@ -71,7 +71,7 @@ export default async function createSandbox(
     console.log(`Creating sandbox with template '${template}'`);
   }
 
-  packageJsonPackage = { main: "/index.html" };
+  packageJsonPackage = packageJsonPackage || { main: "/index.html" };
```

Comment:
The important change, which was done for `static` templates. We lost some data here.